pkg/library/libraryproduces/types.go

```diff
@@ -0,0 +1,38 @@
+package libraryproduces
+
+type ProducedResources struct {
```

Comment:
If you don't mind, could you please explain the purpose of each server? I assume they can be found on an HCP cluster. Which one would OCP developers use?